### PR TITLE
Fix for issues igarape/copcast-server#103 and igarape/copcast-android#174

### PR DESCRIPTION
**Pull request by kamalg19**
igarape/copcast-server#103
- Addresses CC-01-010 Evidence tampering via video upload (Critical)
- Stop mission when state is recording/streaming and date and time is changed to be not automatic

igarape/copcast-android#174
- Fix issue of video upside down for Nexus 5X phones
- Use CameraInfo.orientation (regular phones = 90, nexus 5x phones=270) to calculate display_orientation
- Transpose video based on display_orientation

igarape/copcast-android#198
- Address CC-01-001 Lack of Pinning for HTTPS connections (Medium)
- Pinned intermediate and root certificates. Leaf certificates are not pinned.
- Added root ca certificates for Startcom
- Added intermediate ca certificates: StartCom Class 1 DV Server CA, StartCom Class 1 Primary Intermediate Server CA

### CODE REVIEW

File: app/src/main/java/org/igarape/webrecorder/VideoProducer.java [hunk 1]
```diff
@@ -79,6 +85,38 @@ public VideoProducer(SurfaceHolder surfaceHolder, int videoWidth, int videoHeigh
...
+            rotation = Surface.ROTATION_0;
+        else if (orientationArg == Orientation.BOTTOM)
+            rotation = Surface.ROTATION_180;
+        else if (orientationArg == Orientation.RIGHT)
+            rotation = Surface.ROTATION_90;
+        else
+            rotation = Surface.ROTATION_270;
+
+        int degrees = 0;
+        switch (rotation) {
```
Comment (brunosiqueira) [inline app/src/main/java/org/igarape/webrecorder/VideoProducer.java]: @kamalg19 wouldn't be better to associate the degrees values directly in the previous IF? You are not really using the rotation variable, right?

**Comment by kamalg19** [comment]
Ok, will do change. Earlier was using rotation and forget to put switch code in if block.

**Comment by kamalg19** [comment]
@brunosiqueira Removed rotation variable. Pull request updated. Hope its good for merge now.

**Comment by kamalg19** [comment]
@brunosiqueira 
Added fix for issue igarape/copcast-android#198 to this pull request.

**Pull request merged**